webapp/src/language/en.json

```diff
@@ -1,4 +1,62 @@
 {
+  "sidebar": {
+    "item1": "Block Producers",
+    "item2": "Node Distribution",
+    "item3": "Rewards Distribution",
+    "item4": "Update Node Info",
+    "item5": "About",
+    "item6": "Ricardian Contract",
+    "item7": "Help"
```

Comment:
I think that could be better use "_MENU_OPTION" as suffix and something more specific at the begin like "BLOCK_PRODUCERS", "NODE_DISTRIBUTION", "REWARDS_DISTRIBUTION", "UPDATE_NODE_INFO" and so on